modules/bullet/collision_object_bullet.h

```diff
@@ -246,8 +262,12 @@ class RigidCollisionObjectBullet : public CollisionObjectBullet, public ShapeOwn
 	void set_shape_disabled(int p_index, bool p_disabled);
 	bool is_shape_disabled(int p_index);
 
+	virtual void prepare_object_for_dispatch();
+
 	virtual void shape_changed(int p_shape_index);
-	virtual void reload_shapes();
+	void reload_shapes();
+	bool need_reload_shapes() const { return need_shape_reload; }
+	virtual void do_reload_shapes();
```

Comment:
As far a I can tell, the `need_reload_shapes()` function is never used.

Reply:
Yes, but it's the getter of a variable, it's there for code completeness.